### PR TITLE
Removal of "dbo" prefix from database objects (so db users with different default schema than dbo can be used).

### PR DESCRIPTION
**Pull request by Grizzlik**
https://community.origam.com/t/deployment-script-generator-error-when-same-tables-in-different-schemas-exist/1278

### CODE REVIEW

**Review by JiriPrajz — APPROVED** [review]

File: backend/OrigamArchitect/packages/log4net.2.0.8/lib/net20-full/log4net.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
             <example>
             An example of a SQL Server table that could be logged to:
             <code lang="SQL">
-            CREATE TABLE [dbo].[Log] (
```
Comment (tvavrda) [inline backend/OrigamArchitect/packages/log4net.2.0.8/lib/net20-full/log4net.xml:45]: Are we supposed to change this? It comes from log4net library?

Comment (tvavrda) [inline backend/OrigamArchitect/packages/log4net.2.0.8/lib/net20-full/log4net.xml:45]: Are we supposed to change this? It comes from log4net library?

File: model-tests/model/Root/DeploymentVersion/_origam_root/4.1.origam___commandText___cf5652d6-47c7-44c8-9569-54e7719c2806.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿create type dbo.AsapListValue as TABLE
```
Comment (tvavrda) [inline model-tests/model/Root/DeploymentVersion/_origam_root/4.1.origam___commandText___cf5652d6-47c7-44c8-9569-54e7719c2806.txt:1]: I thought schema was mandatory. Does it work?

**Comment by tvavrda** [comment]
@Grizzlik please review my comments

**Comment by Grizzlik** [comment]
[Testing comment to switch my reviews from pending to submitted]

**Pull request closed without merging**